konduit-serving-examples/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Far as I know we're keeping this in a separate repo anyways?

Reply:
Yes, that makes more sense. I'll shift these to the other repo. They were just here for showcasing some of the API changes

Reply:
As soon as we have maven central distros, we'll shift these to the examples repo